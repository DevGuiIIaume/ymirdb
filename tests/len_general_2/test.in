set a 1
set b a 2
set c b 3
set d 4
set e 5
push d a b c
append d e
forward a
forward b
forward c
forward d
forward e
backward a
backward b
backward c
backward d
backward e
list entries
len a
len b
len c
len d
len e
push a 3 2 1
push b -1 0
append c 4
append e 55
list entries
len a
len b
len c
len d
len e
bye
