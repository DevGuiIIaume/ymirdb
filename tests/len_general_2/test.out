> ok

> ok

> ok

> ok

> ok

> ok

> ok

> nil

> a

> a, b

> a, b, c, e

> nil

> b, c, d

> c, d

> d

> nil

> d

> e [5]
d [c b a 4 e]
c [b 3]
b [a 2]
a [1]

> 1

> 2

> 3

> 8

> 1

> ok

> ok

> ok

> ok

> e [5 55]
d [c b a 4 e]
c [b 3 4]
b [0 -1 a 2]
a [1 2 3 1]

> 4

> 7

> 9

> 23

> 2

> bye
